python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -18,6 +18,9 @@
 import unittest
 import shutil
 import tempfile
+
+import grpc  # type: ignore
```

Comment:
What was wrong for the `mypy` check on this import?

Reply:
GRPC does not provide typing information and we don't have any additional stubs package installed. The easy way for now is to disable it.

```
python/pyspark/sql/connect/client.py:25: error: Skipping analyzing "grpc": module is installed, but missing library stubs or py.typed marker  [import]
```